trackomatic.js

```diff
@@ -281,112 +327,123 @@ function Trackomatic(tracker, config) {
       // if ready has already fired, then just schedule the callback
       // to fire asynchronously, but right away
       if (readyFired) {
-        setTimeout(function() {callback(context);}, 1);
+        setTimeout(function() {
+          callback(context);
+        }, 1);
+
         return;
       } else {
         // add the function and context to the list
-        readyList.push({fn: callback, ctx: context});
+        readyList.push({ fn: callback, ctx: context });
       }
       // if document already ready to go, schedule the ready function to run
-      // IE only safe when readyState is "complete", others safe when readyState is "interactive"
-      if (document.readyState === "complete" || (!document.attachEvent && document.readyState === "interactive")) {
+      // IE only safe when readyState is 'complete', others safe when readyState is 'interactive'
+      if (document.readyState === 'complete' || (!document.attachEvent && document.readyState === 'interactive')) {
```

Comment:
This is a todo on this project — we need to make a solid support browser support decision, because some of these features are IE8-okay, some aren't. I'm voting for IE9+, but that skews data somewhat.